checker/manual/external-tools.tex

```diff
@@ -302,17 +302,29 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
 
 If you use the \href{http://maven.apache.org/}{Maven} tool,
-then you can specify pluggable type-checking as part of your build
-process. This is done by pointing Maven to a script that
-makes Maven use the Type Annotations compiler.
+then you can enable Checker Framework checkers by following the
+instructions below.
+
 These instructions use the artifacts from
 \href{http://search.maven.org/#search\%7Cga\%7C1\%7Corg.checkerframework}{Maven Central}.
 
+See the demonstration \code{examples/MavenExample/} for an example of the use of
+these instructions in a Maven project.  This example can be used to verify that
+the Maven artifacts are retrieved correctly.
```

Comment:
Why does a user want to verify that the Maven artifacts are retrieved correctly?  Can you state what the user cares about in a more direct way?